packages/safe-apps-provider/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@safe-global/safe-apps-provider",
-  "version": "0.15.2",
+  "version": "0.16.0",
```

Comment:
the changeset library should handle all the version changes afterwards, no need to update them manually